### DISCUSSION TITLE
useRequest Suspense

### DISCUSSION THREAD

**Discussion by a896853205** [alibaba/hooks#1362] [Q&A]
想学习一下 ahooks 中如何实现 Suspense 配置的🙏。
但是不是这个 feature 还没上线嘞。在源码中没有找到相关代码。
https://github.com/alibaba/hooks/issues/475

**Comment by brickspert** [comment]
目前没有实现 Suspense 能力。 貌似 React 18 里面 Suspense 已经没了。
